### ISSUE TITLE
runtime terminate in unusual way

### ISSUE THREAD

**Issue by mai80** [4lex4/scantailor-advanced#63]
Each time I get these messages (pictures below), I have to restart the projects all over again under a different project name.  It's getting old.  What is affecting this?  How do we solve it aside from restarting everything?  Thanks.  
![IMG_0310](https://user-images.githubusercontent.com/51537336/59162022-33d93800-8ab8-11e9-8017-7da410c67b1d.jpg)
![IMG_0311](https://user-images.githubusercontent.com/51537336/59162023-35a2fb80-8ab8-11e9-8372-2efdbbd82cd0.jpg)

**Comment by 4lex4** [comment]
I need more information such as at which stage the app crashes and in what circumstances.
Does the problem occur on a particular project or any? If that's the specific project send me the whole project with the project file at 4lex49@zoho.com

**Comment by mai80** [comment]
It happened in 7 of my 15 projects so far (and I plan on doing over 100 projects total).  Always happened during running the output process.  Even directly viewing the page in the output part, where problem was in question have the same result.  I was unable to bypass the problem page to the next one in output either.  The "problem page" thing happened randomly, some projects have this on the very last page.  Other projects have this after running output through the first few of 75-95 pages.  The most recent project have this after the first 20 pages of 80-something.  Do you still want one of my projects?  Thanks!

**Comment by 4lex4** [comment]
Yea, send me one so that I could reproduce the bug and specify the pages in that which cause the crash.

**Comment by mai80** [comment]
I sent you an example a while ago.  I see that you have added "bug" label two days ago.  I hope you can figure out a fix for the bug.  :)  Thanks.

**Comment by 4lex4** [comment]
Yep, got it, thanks for feedback.

**Comment by ymmv99** [comment]
I got a lot of crashes too in the past. The program crashed always with the same images. When I got a crash, I reloaded the project, remove the bad image, ran the output again. When I got another crash, I tried to find the bad image again, remove it. Rinse and repeat. But there was in fact no problem with the source images themselves, A project with only these bad images would output just fine.

What worked was turning off Fill Offcut and Fill Margins in the output stage. I haven't had a single crash since.

**Comment by 4lex4** [comment]
Yep, another effect of the bug it could make the output page blank instead of crashing.
I've already fixed that. Fix will be available in the next version and much sooner in the devel branch which will be updated in about a week or two.
